src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -77,6 +151,66 @@ export const IncidentMap = () => {
           setMapMessage={setMapMessage}
         />
 
+        <StyledViewerContainer
+          topLeft={
+            <TopLeftWrapper>
+              <GPSButton
+                tabIndex={0}
+                onLocationSuccess={(location: LocationResult) => {
+                  const coordinates = {
+                    lat: location.latitude,
+                    lng: location.longitude,
+                  } as LatLngLiteral
+                  fetchLocation(coordinates)
```

Comment:
consider just putting

``` {
                     lat: location.latitude,
                     lng: location.longitude,
                   }```

currently you have a double this type LatLngLiter, which will already by typechecked in the function fetchLocation

Reply:
Removed the as latLingLiteral